dist/common/scripts/scylla_io_setup

```diff
@@ -277,7 +277,54 @@ if __name__ == "__main__":
                 ioconf = open(etcdir() + "/scylla.d/io.conf", "w")
                 ioconf.write("SEASTAR_IO=\"--io-properties-file={}\"\n".format(properties_file.name))
             else:
-                logging.error('This is not a recommended Google Cloud instance setup for auto tuning, running manual iotune.')
+                logging.error('This is not a recommended Google Cloud instance setup for auto local disk tuning, running manual iotune.')
+                run_iotune()
+        elif azure_instance().is_azure_instance():
+            idata = gcp_instance()
```

Comment:
This should be change to `azure_instance()` i guess.

Reply:
fixed